### DISCUSSION TITLE
Why the final link (shown in the address bar) from relative path includes the symbol `#/`?

### DISCUSSION THREAD

**Discussion by xhao79** [docsifyjs/docsify#2406] [Q&A]
How to not include the symbol `#/` in final link shown in the address bar?

I have set
1.  in 'index.html':
    `relativePath: true,`
2. in 'README.md':
    `[test](./test.docx)`

If click on the link 'test', the link shown in the address bar is 
`http://localhost:3000/#/test.docx`
and the page shows '404 - Not found'

If remove the symbol `#/` to following link
`http://localhost:3000/test.docx`
then, it's able to download the 'test.docx'

I want to download the 'test.docx' with out removing the symbol `#/` manually.
How to do it ?

**Comment by paulhibbitts** [comment]
Hi @xhao79 , just seeing this now. Perhaps this issue might provide some pointers? https://github.com/docsifyjs/docsify/issues/1479
